nautobot/dcim/tests/test_filters.py

```diff
@@ -670,6 +670,11 @@ class LocationFilterSetTestCase(FilterTestCases.NameOnlyFilterTestCase, FilterTe
     def setUpTestData(cls):
         common_test_data(cls)
 
+        # Since VLANFactory is not smart enough to create some VLANs without locations we have to manually remove locations form some vlans
```

Comment:
Is this still true/necessary after changing the `minimum` count to 0 in VLANFactory?

Reply:
Still necessary, but I took an alternative route. Instead of clearing the vlans here, I disallowed vlan content type on `Building` and `Floor` LocationType locations.